ldi-core/ldi-common/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldi/rdf/formatter/LdiRdfWriterProperties.java

```diff
@@ -44,6 +43,19 @@ private void setLangFromProperties() {
 	// JSON-LD
 
 	public String getFrameType() {
-		return properties.get(Objects.requireNonNull(FRAME_TYPE));
+		return getProperty(FRAME_TYPE);
+	}
+
+	// Common
+
+	private String getProperty(String key) {
+		return properties.get(stripped(key));
+	}
+
+	static String stripped(String key) {
```

Comment:
If you are adding this logic, can't we then change the keys above to be in line with the documentation?

Reply:
otherwise adding this, makes little sense

Reply:
can't this be private? This does not seem difficult to test